manager/controllers/default/resource/create.class.php

```diff
@@ -214,18 +222,10 @@ public function getDefaultTemplate() {
                 case 'sibling':
                     if (!empty($this->parent->id)) {
                         $c = $this->modx->newQuery('modResource');
-                        $c->where(array('parent'=>$this->parent->id, 'context_key'=>$this->ctx));
+                        $c->where(array('parent' => $this->parent->id, 'context_key' => $this->ctx));
                         $c->sortby('id', 'DESC');
-                        $c->limit(1);
-                        $siblings = $this->modx->getCollection('modResource', $c);
-                        if (!empty($siblings)) {
-                            foreach ($siblings as $sibling){
-                                $defaultTemplate = $sibling->get('template');
-                            }
-                        }else{
-                            if (!empty($this->parent->id))
-                                $defaultTemplate = $this->parent->get('template');
-                        }
+                        $lastSibling = $this->modx->getObject('modResource', $c);
+                        $defaultTemplate = !empty($lastSibling) ? $lastSibling->get('template') : $this->parent->get('template');
```

Comment:
You have missed the `!empty($this->parent->id)` check in the else branch. Is it safe to remove that?

Reply:
@Jako Look at the parent if block and everything will become clear.

Reply:
Then there is a bug or a glitch in the code. In root an automatic template assignment is not possible for siblings.

Reply:
That's right. 
If you look at the controller code more carefully you will have a lot of questions.

The most harmless question
```
// Default id is 1
$this->modx->getOption('default_template', null, 1)
...
// But here the default id is already 0
$this->context->getOption('default_template', 0, $this->modx->_userConfig);
```
The code of the `getDefaultTemplate` method is strange. What is the first line for? And so on.